src/app/dataValidator/helpers/dataLoader/readScenario.js

```diff
@@ -2,21 +2,36 @@ const fs = require('fs');
 const path = require('path');
 const { validateData } = require('../validators/validateData');
 const { countScenarios } = require('../../utilities/countScenarios');
+const readdirSync = require('./readdirSync');
+
+const ifDirectoryValidateNestedFiles = fullFileName => {
+  const fileStats = fs.statSync(fullFileName);
+
+  if (fileStats.isDirectory()) {
+    readdirSync.readdirSync(fullFileName);
```

Comment:
This does not use destructing assignment as it creates the following error: 
```
TypeError: readdirSync is not a function
```

This is due to the recursive nature of the files.